blog/2025-04-15-oracle-wars.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Oracle Wars - Exploring Real-Time Oracle Behavior and Push-Based Feeds
+sidebar_label: Oracle Wars - Exploring Real-Time Oracle Behavior and Push-Based Feeds
+slug: /oracle-wars
+---
+
+<img src="/blog-assets/oracle-wars-1.png" alt="Cover Image Oracle Wars" width="100%"/>
+
+<!--truncate-->
+
+Oracles are the unsung heroes of DeFi. They connect blockchains to real-world data, enabling smart contracts to interact with off-chain events like asset prices (USDC, ETH), market rates, or even real-time sports scores. But while the concept of oracles is often discussed in theoretical terms, visualizing how they behave in practice can offer a new level of clarity, especially for developers building apps on-chain.
+
+Enter [Oracle Wars](https://www.oraclewars.xyz/), a live feed of on-chain oracle data that compares how different providers behave in real-time. It helps developers visualize and gain a better understanding of how oracles actually function under different market conditions, so they can design more reliable and secure smart contracts.
+
+⭐ **Please note**: The dashboard shown above is no longer live on the site, but the platform remains focused on delivering valuable insights into oracle performance. Oracle Wars is an educational and experimental platform designed to showcase data from various oracle providers, with the goal of improving the understanding of oracle behavior through real-time data analysis, which is constantly evolving and being updated.
```

Comment:
Hey @Jordy-Baby, you should include something like "The site is currently showing live megaeth updates, and not this dashboard" etc... Otherwise this alone doesn't make much sense? What is it actually showing then